common/utils/mission.js

```diff
@@ -144,7 +149,7 @@ export function computeMissionStats(m, users) {
       endTime,
       endTimeOrNow,
       service: endTimeOrNow - startTime,
-      totalWorkDuration,
+      totalWorkDuration: totalWorkDuration - transferDuration,
```

Comment:
Are we sure the transferDuration doesn't count in totalWorkDuration?

Reply:
From my understanding: transfer should impact Amplitude, but not Temps de Travail.
let's say we have 05h-14h(transfer)/14h-15h(drive)
with old version we see amplitude=10h/tempsdetravail=10h in the mission card
with new version we see amplitude=10h/tempsdetravail=1h
which is what I expect